scripts/components/SongCards.js

```diff
@@ -101,7 +101,7 @@ class SongCards extends Component {
 
     renderSongs(start, end) {
         const chunk = 5;
-        const {authed, dispatch, playlist, playlists, playingSongId, songs, users} = this.props;
+        const {authed, dispatch, playlist, playlists, playingSongId, songs, users, toggleStats} = this.props;
```

Comment:
Yeah, this codebase isn't perfect (obviously). I actually started out with having all my props at the root container level and passed them down to the children. I noticed that since `currentTime` was changing every second, this caused the entire component tree to re-render. This was when I realized the usefulness of `mapStateToProps` (at least for my use case). There are some places that I haven't gotten around to refactoring ;)